docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -30,4 +30,399 @@ Risk factor mediation
+
+where :math:`M` is the set of mediators that mediate the relationship between risk :math:`r`
+and outcome :math:`o` and :math:`\Delta_\text{r→m}` is the (assumed linear) increase in :math:`m`
+caused by a one-unit increase in :math:`r`.
+
+.. note::
+  This ignores the concept of a TMREL below which the RR does not apply and risk
+  is flat.
+  This seems to be ignored throughout the GBD's descriptions of its mediation methods, although
+  some risks that are mediators in GBD, such as BMI, clearly have non-zero TMRELs.
+
+.. note::
+  :math:`\Delta_\text{r→m}` only makes sense if both the risk and the mediator are continuous values.
+  Since all of the mediation we have modeled so far has been between continuous
+  risk factors, we haven't yet explored how to make different assumptions (neither has GBD).
+
+Approach 1: consistent with RR application, inconsistent with GBD mediation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this approach, we work backwards from the result reported by GBD (the mediation factor)
+to an underlying data point we can use to create an internally-consistent set of causal effects.
+It is an "undo and redo" approach: we undo the calculations GBD did in additive space to infer
+the data they used as inputs, then do a completely different calculation on those data in multiplicative space.
+To do this, we have to know whether the mediation factor for the pathway in question was calculated
+with the adjusted RR method ("Comparing crude RR versus mediator-adjusted RR"
+in `the GBD risk factors methods appendix [supplementary appendix 1] <https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30752-2/fulltext#supplementaryMaterial>`_) or
+the delta method ("Estimating the mediation factor by pathway of the effect").
+
+.. todo::
+
+  Where should someone look to find this out?
+
+Adjusted RR method
+^^^^^^^^^^^^^^^^^^
+
+In this case, GBD's data were a pair of RRs for the risk-outcome pair,
+one adjusted for the mediator and one not.
+We presume that the unadjusted RR is (approximately) equal to the total RR
+GBD reports for the risk-outcome pair.
+Therefore, we manipulate this equation from the GBD methods appendix:
+
+.. math::
+
+  MF = \frac{RR_\text{r→o,unadjusted for m} - RR_\text{r→o,adjusted for m}}{RR_\text{r→o,unadjusted for m} - 1}
+
+to solve for :math:`RR_\text{r→o,adjusted for m}`:
+
+.. math::
+
+  RR_\text{r→o,adjusted for m} = RR_\text{r→o,total} - MF * (RR_\text{r→o,total} - 1)
+
+:math:`RR_\text{r→o,adjusted for m}` is the underlying data point, but it
+isn't particularly useful to us, as it doesn't necessarily show up
+anywhere in our causal diagram.
+We can use it to calculate a delta, an (assumed linear) increase in the mediator
+per unit increase in the mediated risk:
+
+.. math::
+
+  \Delta_\text{r→m} = \frac{log(RR_\text{r→o,total}) - log(RR_\text{r→o,adjusted for m})}{log(RR_\text{m→o,total})}
```

Comment:
Where does this equation come from? It looks like maybe it comes from the "equation describing how RRs are applied" above (if so, you could add an equation number and reference to link back to it), with the assumption that there is only a single mediator $m$, and $RR_\text{r→o,direct} = RR_\text{r→o,adjusted for m}$. Is that right? What do you do if there are multiple mediators?

Reply:
Good questions! I think I forgot to include how I got here. I believe the thinking was that

$$
RR_\text{r→o,adjusted for m} = \frac{RR_\text{r→o,total}}{\prod_{m2 \in (M-\\{m\\})} (RR_\text{m2→o})^{\Delta_\text{r→m2}}}
$$

so this equation would stay the same regardless of the number of mediators.

I'll take a pass at fleshing this out.